doc/dev/release-checklists.rst

```diff
@@ -145,7 +138,9 @@ After dev freeze
 ================
 
 - [ ] create branch for new release
-- [ ] remove release notes for release branch: see also 33d63c32cbf81ed81ce2185063011b98846a0d44
+- [ ] remove ``doc/releases/*.rst``. This should leave behind ``doc/releases/releases.yml`` which is used for doc building purposes.
```

Comment:
I liked that `see also 33d63c32cbf81ed81ce2185063011b98846a0d44` -- there is a lot of additional detail in the commit message there.  I'd suggest appending
> See commit 33d63c32cbf81ed81ce2185063011b98846a0d44 ("doc: remove release notes for release branch") for details.

here.

Reply:
done, thanks